### DISCUSSION TITLE
Is there an easy way to disable collision between objects?

### DISCUSSION THREAD

**Discussion by randomanimedude** [jrouwe/JoltPhysics#559] [Q&A] [answered]
I need to disable collision between object and wall and enable it back as soon as object stops colliding with wall. In other words, I need the object to pass through wall and be locked by it on the way back.
Tricky part is, there are multiple objects that still must collide between themselves, but only some of them must ignore collision with wall.
I don't believe GroupFilterTable can help me with that.

**Comment by jrouwe** [comment, marked as answer]
ContactListener::OnContactValidate would be the place for this. You can reject a contact there based on your own logic.
